.github/workflows/e2e_tests.yaml

```diff
@@ -0,0 +1,57 @@
+name: E2E tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'branch to run tests from, default: main'
+        required: false
+        type: string
+      environment:
+        type: choice
+        description: Environment
+        options:
+          - stage
+          - prod
+
+env:
+  branch: ${{ inputs.branch || 'main' }}
+  ols_url: ${{ inputs.environment == 'stage' && 'https://stage.ai.ansible.redhat.com/chat' || 'https://c.ai.ansible.redhat.com/chat' }}
```

Comment:
Dear reviewer, please consider this:

For the URLs here I used the ai connect chat urls but what we need here is the exposed urls of the chatbot service itself. if these current URLs give us what we need right now then it should be enough but otherwise for running the e2e tests as designed we need a route for each environment.

Reply:
if this is sth we can discuss only when @TamiTakamiya returns, what I suggest is: let's merge this PR and that way we will be able to merge: https://github.com/ansible/ansible-wisdom-ops/pull/1179 and  https://github.com/ansible/ansible-wisdom-ops/pull/1180

Reply:
@jameswnl ^^

Reply:
we don't expose the chatbot service to the public. so only though the wisdom-service